.travis.yml

```diff
@@ -5,10 +5,13 @@ cache: yarn
 
 node_js:
   - 6
-
-
+  - 7
+  - 8
+before_install:
+  - cp .env.example .env
+  - cp config.js.example config.js
 install:
-  - npm install
-  - npm install -g nodeunit
+  - yarn
```

Comment:
Just out of curiosity, why choose yarn over npm?  I personally don't have a preference, just curious.

Reply:
In retrospect, I'm also curious about this.

Reply:
Caching.

Reply:
Yarn keeps a cache on travis.

Reply:
Anecdotal, but I have also been using yarn for a while now and found it much faster and predictable than npm. The yarn lock file helps, but npm has that now as well with v5. The speed is real though.

Reply:
Yeah builds on travis are WAY quicker...esp. due to the caching.